tdrs-frontend/zap-scanner.sh

```diff
@@ -20,12 +25,25 @@ echo "================== OWASP ZAP tests =================="
 # Ensure the reports directory can be written to
 chmod 777 $(pwd)/reports
 
-# Run the ZAP scan and generate an HTML report
-docker-compose run zaproxy zap-full-scan.py \
-	-t http://tdp-frontend \
-	-m 5 \
-	-z "${ZAP_CONFIG}" \
-	-r owasp_report.html | tee /dev/tty | grep -q "FAIL-NEW: 0"
+# check if running in circle CI
+
+
+if [ "$ENVIRONMENT" = "circle" ]; then
+    echo "Config file $ENVIRONMENT"
+    docker-compose run zaproxy zap-full-scan.py \
+                   -t http://tdp-frontend/ \
+                   -m 5 \
+                   -z "${ZAP_CONFIG}" \
+                   -c "zap.conf" \
+                   -r owasp_report.html | tee /dev/tty | grep -q "FAIL-NEW: 0"
```

Comment:
I see these alerts being ignored in the output from the ZAP scan but they still show up in the HTML version of the report. Perhaps there is a flag missing here to get that output to work in both places?

Reply:
Did some digging on this and found that it is currently an expected but undesired behavior with the provided ZAP scripts:
https://github.com/zaproxy/zaproxy/issues/5212

It looks like it may be possible to disable these alerts entirely using the ZAP API: https://groups.google.com/g/zaproxy-users/c/p1aXvO6oWu4/m/xUmhPwIDCmIJ

Reply:
I was looking into this and I think we should be able to do it, but I think its a bit more of a lift than needs to be a part of this ticket to do it.

Reply:
Agreed, once we are scanning against deployed sites this should be irrelevant as well.

Reply:
We could capture this follow on work with: https://github.com/raft-tech/TANF-app/issues/1013

Reply:
> We could capture this follow on work with: #1013

@jtwillis92 @riatzukiza agree here. let's capture this in the follow-on work. I think we'd want to address what's contained in the html report before the next release. OCIO has suggested in prior reviews to exclude false-positives in these reports. and it's reasonable to expect their review prior to the next release.

Reply:
Opened https://github.com/raft-tech/TANF-app/issues/1032 to track only the scans against deployed sites.